modules/openy_gc_auth/modules/openy_gc_auth_custom/openy_gc_auth_custom.install

```diff
@@ -126,3 +126,13 @@ function openy_gc_auth_custom_update_8006() {
     'migrate_plus.migration_group.gc_auth',
   ]);
 }
+
+/**
+ * Add value for require_email_verification setting.
+ */
+function openy_gc_auth_custom_update_8007(&$sandbox) {
+  \Drupal::configFactory()
+    ->getEditable('openy_gc_auth.provider.custom')
+    ->set('require_email_verification', 0)
```

Comment:
It should work but it is different to what we use by default in the Open Y system.
I like it more because it is more readable.

Reply:
Upgrade tool param update does not work on our builds and demo because of this:

```php
  /**
   * {@inheritdoc}
   */
  public function update($config, $config_name, $param) {
    // Get base storage config.
    if (!file_exists($config)) {
      $this->logger->error($this->t('File @file does not exist.', ['@file' => $config]));
      return;
    }
```

VY module is a symlink in the modules directory, so file not exist. But on standard installations, it will work fine. Let's keep the current solution, this is not critical to avoid an upgrade tool in the case of the current config.